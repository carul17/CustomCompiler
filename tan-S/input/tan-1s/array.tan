main {
	var a := ['a', 'b', 'c'];
	print a \n;
	
	var b := [1.1, 2.2, 3.3];
	print b \n;
	
	var c := [1,2,4,5];
	print c \n;
}