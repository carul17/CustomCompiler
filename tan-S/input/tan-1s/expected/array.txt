[a, b, c]
[1.100000, 2.200000, 3.300000]
[1, 2, 4, 5]
